wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -313,7 +313,7 @@ body.page:not(.twentyseventeen-front-page) .entry-title {
 
 .navigation-top .current-menu-item > a,
 .navigation-top .current_page_item > a {
-	color: var(--wp20--color--text-primary);
+	color: var(--wp20--color--text);
```

Comment:
Don't worry too much about the navigation. It needs some rework for it to match the designs on mobile. I'll take care of it in #7.